src/routes/docs/tutorials/nextjs-ssr-auth/step-8/+page.markdoc

```diff
@@ -4,7 +4,7 @@ title: All set
 description: Add authentication to a Next.js project using Appwrite.
 step: 8
 ---
-If you want to see the complete source code with styling, see the [demos-for-react](https://github.com/appwrite/demos-for-react/tree/main/nextjs/server-side-rendering) repository.
+If you want to see the complete source code with styling, see the [demos-for-react](https://github.com/appwrite/demos-for-react) repository.
```

Comment:
It would be good to link to the specific folder: https://github.com/appwrite/demos-for-react/tree/master/nextjs/server-side-rendering to save people from digging around.